src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/hooks/usePrepareSaveEditCycleData.tsx

```diff
@@ -87,9 +89,9 @@ export const usePrepareSaveEditCycleData = () => {
 
   const editingFundingCycleData: V2V3FundingCycleData = {
     duration: BigNumber.from(durationSeconds),
-    weight: BigNumber.from(
-      issuanceRateFrom(formValues.mintRate?.toString() ?? '0'),
-    ),
+    weight: mintRateHasDiff
+      ? newMintRate
+      : reduxConfig.editingFundingCycleData.weight,
```

Comment:
if no change, shouldn't this be `WEIGHT_UNCHANGED`?

Reply:
Good catch, but no - this is eventually passed into this function inside `useReconfigureFundingCycle`. As it works now, passing `WEIGHT_UNCHANGED` (`0`) as `newFundingCycleWeight` would be interpreted as the user wanting a weight `0`, in which case we pass `1` to the contracts. 

<img width="771" alt="Screen Shot 2023-09-09 at 3 01 57 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/4b0d3540-d0c2-469c-ae1e-aa326de5d121">